public/app/features/alerting/unified/components/silences/SilencesEditor.tsx

```diff
@@ -97,24 +95,24 @@ const getDefaultFormValues = (searchParams: URLSearchParams, silence?: Silence):
   }
 };
 
-export const SilencesEditor = ({ silence, alertManagerSourceName }: Props) => {
+export const SilencesEditor = ({ silenceId, alertManagerSourceName }: Props) => {
+  const history = useHistory();
+  const [getSilence, { data: silence, isLoading: getSilenceIsLoading }] =
```

Comment:
I think it might be beneficial to add a comment for why we're going with a lazy query instead of a regular one (for it to work with blank forms)